src/instances/Collection/index.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { describe, it, expect } from 'vitest'
+import { CollectionInstance } from './index'
+import { makeClient } from '../../client'
+import { collectionInfo, collectionItems } from '../../../mocks/fixtures'
+
+describe('CollectionInstance methods', () => {
+  it("should return the proper collection object (using collection('collectionName'))", async () => {
```

Comment:
```suggestion
  it("should return the proper collection object)", async () => {
```
I think it's unnecessary to add the function in the description as this file only has tests related to that function. Also the function name is already in the import